care/facility/migrations/0105_auto_20200425_1446.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 2.2.11 on 2020-04-25 09:16
+
+from django.db import migrations, models
+import uuid
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [standard import "import uuid" should be placed before "from django.db import migrations, models"](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5419355)